### PR TITLE
Fix finalizers

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: glib/Object.cs [hunk 1]
```diff
@@ -95,8 +95,6 @@ public virtual void Dispose ()
 				Console.WriteLine ("Exception while disposing a " + this + " in Gtk#");
```
Comment (alanmcgovern) [inline glib/Object.cs]: Can we get a description of what was broken, why it was broken and how this fixes it? Future alan will be glad that the information was documented in the commit message ;)

Comment (alanmcgovern) [inline glib/Object.cs]: Thats for the `[GLib] Fix GObject finalizers to finally work.` commit

Comment (Therzok) [inline glib/Object.cs]: Done.

Comment (alanmcgovern) [inline glib/Object.cs]: Can we remove `bool owned = true;` now that the field is unused?

Comment (Therzok) [inline glib/Object.cs]: Definitely.

**Pull request merged**